### PR TITLE
52 update dockerfiles based on quarkus latest

### PR DESCRIPTION
**Pull request by joewhite101**

### CODE REVIEW

File: generators/server/files.js [hunk 1]
```diff
@@ -343,7 +343,7 @@ const serverFiles = {
         {
             path: DOCKER_DIR,
             condition: generator => generator.buildTool === 'maven',
```
Comment (avdev4j) [inline generators/server/files.js:345]: I wonder why we added this condition, we should add them for gradle too, isn't us?

File: generators/server/templates/quarkus/src/main/docker/Dockerfile.fast-jar.ejs [hunk 2]
```diff
@@ -0,0 +1,56 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the docker image run:
+#
+# mvn package
```
Comment (avdev4j) [inline generators/server/templates/quarkus/src/main/docker/Dockerfile.fast-jar.ejs:6]: If we will support gradle too we have to change this according to the choosen option

File: generators/server/files.js [hunk 3]
```diff
@@ -343,7 +343,7 @@ const serverFiles = {
...
-            templates: ['Dockerfile.jvm', 'Dockerfile.native']
+            templates: ['Dockerfile.jvm', 'Dockerfile.native', 'Dockerfile.fast-jar']
```
Comment (avdev4j) [inline generators/server/files.js:346]: When the Maven/Gradle condition will be removed, I suggest to add those files to the expect-files.js to improve our test coverage.

File: generators/server/files.js (hunk 1 above)
Comment (danielpetisme) [reply generators/server/files.js:345]: This is my fault... 
The first release of this blueprint was supporting Maven only so I've added this condition.
Make not sense anymore

Comment (avdev4j) [reply generators/server/files.js:345]: It's made sense for sure. It's just that I don't want to ask people to change something I'm not sure. 

If you are ok @joewhite101 we could use this PR to remove this condition. I think there is nothing special to do, unless remove the condition and test.

**Comment by joewhite101** [comment]
@avdev4j Works for me. Remove the condition, update the templates, and let's test.

**Review by avdev4j — APPROVED** [review]

**Pull request merged**